db/node_jobs.go

```diff
@@ -57,14 +57,19 @@ func GetNodeJobValidatorInfos(job *types.NodeJob) ([]types.NodeJobValidatorInfo,
 	if err != nil {
 		return nil, err
 	}
+	jobStatus := "Pending"
+	switch job.Status {
+	case types.SubmittedToNodeNodeJobStatus:
+		jobStatus = "Subbmitted to node"
```

Comment:
Typo? guess this should be **Submitted**

Reply:
Off course, fixed ;-)